.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,49 @@
-- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+<!--
+Thank you for sending the PR! 
+Please fill the applicable details below
+Happy contributing!
+-->
 
-- **Why was this change needed?** (You can also link to an open issue here)
+### What kind of change does this PR introduce? <!-- Mark The Boxes -->
 
-- **Other information**:
+- [ ] Bug
+- [ ] Feature
+- [ ] Docs
+- [ ] Other(s)
+
+
+<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
+
+### Fixes Issue
+
+<!-- Remove this section if not applicable -->
+
+<!-- Example: Closes #31 -->
+
+### Why was this change needed?
+
+<!-- Explain all the changes in the PR and tell what is the use of this change. -->
+
+<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
+<!--
+[x] - Correct; marked as done
+[X] - Correct; marked as done
+[ ] - Not correct; marked as **not** done
+-->
+
+### Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->
```

Comment:
I agree with David, that this checklist is not needed at the moment

Reply:
I'll fix it and remove it

Reply:
I have removed it @ainouzgali